README.md

```diff
@@ -1 +1,189 @@
-# Webpack-BoilerPlate
\ No newline at end of file
+![](https://img.shields.io/badge/MazeFlix-red)
+
+# MazeFlix
+
+> MazeFlix is an entertainment hub where you can find the latest and your favorite TV shows. You can view the details of the show, like a show or even comment. You can also see other comments which might give you a general idea about what people think about the show.
+> It is built with HTML, CSS and JavaScript with data from the TvMaze API(tv show data) and the Involvement API(likes and comments)
+
+## Project Presentation
+[Presentation Link](https://drive.google.com/drive/folders/1F7nq4FhOPCONHq84RJJ6RLaSfYrdU53a)
+
+## Mini Demo
+
+![screenshot](./src/assets/images/maze-flix.gif)
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages (HTML, CSS, JavaScript)
+- Frameworks (None)
+- Technologies used 
+  
+  ``` bash
+  - Webpack(Code Bundlng et al)
+  - Jest(for testing)
+  - Git(version control)
+  - ESLint(JavaScript linting)
+  - WebHint(linting tool)
+  - Stylelint(style linting)
+  ```
+
+## Live Demo
+
+[Website Link](https://anselemodims.github.io/MazeFlix/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+ - A text editor(preferably Visual Studio Code)
+### Install
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
+### Usage
+#### Clone this repository
+
+```bash
+$ git clone git@github.com:AnselemOdims/MazeFlix.git
+$ cd MazeFlix
+```
+#### Run project
+
+```bash
+$ npm install
+$ npm run start # this will make webpack watching for your changes in code
+```
+
+#### Open page in browser
```

Comment:
- [ ] Please mention the command on how to test the application. [ Hint: `npm test`]